android/src/main/java/com/pspdfkit/views/PdfView.java

```diff
@@ -309,7 +334,91 @@ public void exitCurrentlyActiveMode() {
 
     public void saveCurrentDocument() {
         if (fragment != null) {
-            fragment.save();
+            try {
+                if (document.saveIfModified()) {
+                    // Since the document listeners won't be called when manually saving we also dispatch this event here.
+                    eventDispatcher.dispatchEvent(new PdfViewDocumentSavedEvent(getId()));
+                }
+            } catch (IOException e) {
+                // We have no easy way to notify react about this so just swallow it for now.
```

Comment:
Would this require another event type?

Reply:
Yes, we could expose a `onDocumentSavingFailed` callback on our react component and expose it there.

Reply:
Does iOS also have possible saving errors that we could notify of that way @radazzouz ?

Reply:
Yes. We have [-pdfDocument:saveDidFailWithError:](https://pspdfkit.com/api/ios/Protocols/PSPDFDocumentDelegate.html#/c:objc(pl)PSPDFDocumentDelegate(im)pdfDocument:saveDidFailWithError:) on iOS, but we didn't expose it as a separate React Native callbackl yet.

@irgendeinich how about we use `onDocumentSaveFailed` instead?

Reply:
@radazzouz 
Yes, let's expose `onDocumentSaveFailed`

Reply:
Awesome! Done in https://github.com/PSPDFKit/react-native/pull/99/commits/08b97380e1ddd0f0de54857a6a294e3f560fbb53

@irgendeinich, @steviki since you already approved #99 before I committed the `onDocumentSaveFailed `, can you please re-review this PR?